app/src/main/res/layout/view_location_list_preference.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android" android:orientation="vertical" android:layout_width="match_parent"
+  android:layout_height="match_parent">
+  <SearchView
+    android:id="@+id/filter"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:iconifiedByDefault="false"
+    android:hint="@string/filter_gdg_by_name"/>
+  <org.gdg.frisbee.android.view.FilterListView
+    android:id="@android:id/list"
+    android:layout_width="match_parent"
+    android:layout_height="150dp"/>
```

Comment:
I was searching where the height is coming from. Although the height seems ok, the normal `ListPreference` fills the whole screen height. What do you think?